pkg/kubelet/dockertools/convert.go

```diff
@@ -27,6 +28,19 @@ import (
 // This file contains helper functions to convert docker API types to runtime
 // (kubecontainer) types.
 
+func mapStatus(status string) kubecontainer.ContainerStatus {
+	// Parse the status string in docker.APIContainers. This could break when
```

Comment:
This *might* break because we are parsing a human readable string. On the other hand, it should be easy to verify when upgrading docker.